src/library_postprocessing.rs

```diff
@@ -503,3 +504,14 @@ impl Library {
         }
     }
 }
+
+fn create_fixed_array_c_type_from_inner(inner_c_type: &str) -> String {
+    let mut c_type = inner_c_type.trim_end();
+    // removing last *
+    if let Some(pos) = c_type.rfind("*") {
```

Comment:
You're sure this is a good idea?

Reply:
Not fuly. But before in .gir files in "array" tag was element type while in inner "type" - array type